velox/functions/sparksql/tests/DateTimeFunctionsTest.cpp

```diff
@@ -301,6 +301,25 @@ TEST_F(DateTimeFunctionsTest, unixTimestampCustomFormat) {
   EXPECT_EQ(
       std::nullopt,
       unixTimestamp("2022-12-12 asd 07:45:31", "yyyy-MM-dd 'asd HH:mm:ss"));
+
+  const auto unixTimestamp1 = [&](std::optional<Timestamp> timestamp,
```

Comment:
It would be better to add separate tests for timestamp and date inputs as the test name 'unixTimestampCustomFormat' indicates it focuses on string.